<?xml version="1.0"?>
<launch>
   <node name="rona_obstacle_handler_node" pkg="rona_obstacle" type="rona_obstacle_handler_node" output="screen">
      <param name="pub_marker_topic"             type="string" value="obstacle_fix_marker"   />
      <param name="pub_add_obstacle_topic"       type="string" value="add_obstacle"          />
      <param name="pub_rm_obstacle_topic"        type="string" value="rm_obstacle"           />
      <param name="sub_obstacle_topic"           type="string" value="obstacle"              />
      <param name="map_frame"                    type="string" value="map"                   />
      <param name="robot_radius"                 type="double" value="0.4"                   />
      <param name="duration_obstacle_valid"      type="double" value="60 "                   />
   </node>                                                                               
</launch>